### PR TITLE
Fix uninitialized members in Wasm2AsmBuilder::Flags

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by kripken** [comment]
How about initializing those fields in the struct declaration? (Then it's just one place instead of 2, and would work in other users in the future too.)

**Comment by sbc100** [comment]
Done

File: src/wasm2asm.h [hunk 1]
```diff
@@ -109,6 +109,7 @@ class Wasm2AsmBuilder {
 
 public:
   struct Flags {
+    Flags() : debug(false), pedantic(false) {}
     bool debug: 1;
```
Comment (binji) [inline src/wasm2asm.h]: Not that anyone asked, but IMO this shouldn't be using bitfields either. :-)

Comment (kripken) [reply src/wasm2asm.h]: I agree.

Also personally I'd prefer `bool debug = false;` (init in the declaration). But lgtm either way.

Comment (sbc100) [reply src/wasm2asm.h]: What?  You want to waste almost 8 entire bytes for every command line run!  You know thats almost 64 bits right?

**Pull request merged**